src/app/profile/actions.ts

```diff
@@ -3,11 +3,18 @@
 import { revalidatePath } from "next/cache";
 import { redirect } from "next/navigation";
 import prisma from "@/lib/db/prisma";
+import { isReciever } from "@/src/actions/actions";
 
 export const DeleteInvite = async (inviteId: string) => {
+  const invite = await prisma.invite.findUnique({ where: { id: inviteId } });
+
+  if (!invite) return;
+
+  if (!(await isReciever(invite.userEmail))) return;
+
   await prisma.invite.delete({ where: { id: inviteId } });
-  revalidatePath("/");
-  redirect("/");
+  revalidatePath("/profile");
```

Comment:
czy to nie zrobi revalidate każdemu użytkownikowi?

czy może własny profil nie powinien też mieć id użytkownika w środku?

chyba że głupoty gadam i nie wiem jak revalidate działa